pyproject.toml

```diff
@@ -67,7 +67,7 @@ test = [
     "django>=4,<6",
     "numpy~=1.26.0",
     "flask_wtf~=1.2.0",
-    "fickling~=0.1.0",
+    "fickling~=0.1.0,>=0.1.3",
```

Comment:
any reason not to just make it `>=0.1.3"`?

Reply:
I think it's safer to keep it explicitly pinned to a particular minor version for now.